charts/k6-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.0.11"
 description: A Helm chart to install the k6-operator
 name: k6-operator
-version: 1.2.0
+version: 2.0.0
```

Comment:
Why do you think it should be a breaking change? This seems like a "pure" addition: old deployments should continue to work as is. I.e. why not 1.3.0?

FYI, I'll need to test it a bit and probably return to this PR ~ tomorrow.

Reply:
Thanks for sharing your experience with versioning of other Helm charts. The chart in k6-operator was added very recently and we're figuring out the "right way" still :sweat_smile: 
Let's go with major bump this time indeed :+1: